libs/ledger-live-common/src/families/ethereum/synchronisation.ts

```diff
@@ -77,6 +77,27 @@ export const getAccountShape: GetAccountShape = async (infoInput, { blacklistedT
 
   // transform transactions into operations
   let newOps = flatMap(txs, txToOps({ address, id: accountId, currency }));
+
+  // keep the date of the pending operations that appears in newOps
+  const pendingOperationsDate = {};
```

Comment:
I don't understand why we need to do this :(

The pendingOperationsDate map is reset on each sync, right ?

Reply:
yes, the pendingOperationsDate map is reset on each sync. (I rename the "pendingOperationsDate" variable now)
I don't see any better solution. To scan all the operations in initialAccount, it is not very time consuming. 
In fact, we have already another place to scan all operations on each sync. "mergeOps".
I wanted to reuse it but it seems to be very difficult.